src/components/MainVisualization.js

```diff
@@ -42,54 +42,68 @@ function MainVisualization({ coverageData, className }) {
   return (
     <div className={`grid grid-cols-3 gap-5 flex-auto ${className}`}>
       <SectionCard
-        className="h-48 col-span-3"
+        className={selectedSection === 'Overall' ? 'h-48 border-2 border-black' : 'h-48'}
+        colspan="col-span-3"
         header={<p className="font-sans font-bold text-4xl">Overall mCODE Coverage</p>}
         text={<p className="text-s text-gray-400">{patient.possible} Patients</p>}
         gaugeSize="h-44 w-44"
         percentage={overall.percentage}
         color="#000000"
+        onClick={() => setSelectedSection('Overall')}
       />
       <SectionCard
+        className={selectedSection === 'Patient' ? 'border-2 border-patient' : ''}
```

Comment:
Anywhere we're referencing these strings we should be using ids exported from `coverageSectionIds`